### ISSUE TITLE
Chapter 5 – Exercise "running elixir through CLI"

### ISSUE THREAD

**Issue by jonathan-soifer** [radar/joyofelixir#110]
Hi!

Good afternoon 😊

At the end of Chapter 5, the last exercise is about saving any of the code to its own file and experimenting with the CLI

![Screenshot 2022-04-09 at 15 44 28](https://user-images.githubusercontent.com/8316698/162576914-12e14741-393a-4ed3-9290-a8bf1f343190.png)

And that is awesome! Although, while trying to run this on the latest elixir version (`1.13.4` at the moment of this message) – **I get no output to the console.**

Because of that, I had to change my code, from:

![Screenshot 2022-04-09 at 15 47 47](https://user-images.githubusercontent.com/8316698/162577025-448675d1-c183-44ff-a382-71dd4e08af35.png)

To:

![Screenshot 2022-04-09 at 15 48 12](https://user-images.githubusercontent.com/8316698/162577033-73a1a135-7b25-4745-ad05-d55a33845b02.png)

*TL;DR:* It seems like we need to use `IO.puts/1` in order to complete this exercise in a way that a "newbie" would feel comfortable (by seeing the output to the console)

Unless I am mistaken, I believe this was not taught before and this might demotivate some people.

Would you consider changing this?
